switchtheme.module

```diff
@@ -158,11 +161,17 @@ function switchtheme_custom_theme($return_single = TRUE) {
  */
 function switchtheme_block_info() {
   $blocks['switch_form'] = array(
-    'info' => t('Switch theme'),
+    'info' => t('Switchtheme Form'),
```

Comment:
You should not capitalize words in labels unless referring to module/theme/layout names.

```suggestion
    'info' => t('Switchtheme form'),
```

Reply:
Or perhaps:

```suggestion
    'info' => t('Switchtheme: Form'),
```